### PR TITLE
terminal changed

### LINKED ISSUE #1967: [Learn] Change the look of terminal in learn-ng homepage
**Current Behavior**
The terminal present in the [Learn-NG](https://layer5.io/learn-ng) is not looking good.

Screenshot 2021-07-05 at 2 57 19 PM

**Desired Behavior**
Make the terminal look good and display relevant content in it.

Inspiration can be taken from this [terminal](https://layer5.io/service-mesh-management/meshery/getting-started)
Infact, it can be reused 🤔 

Screenshot 2021-07-05 at 2 59 40 PM

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by chetak123**
Signed-off-by: Ayushman Mishra <ayushvidushi01@gmail.com>

**Description**
Screenshot from 2021-07-08 13-04-57

This PR fixes #1967 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit bbd84bb587636f052801e0d5d637877ab7d129a4 at: https://60e6afdf709419a434ef6832--layer5ng.netlify.app

**Review by sudo-NithishKarthik — COMMENTED** [review]
Awesome @chetak123  🙌🏽 

Can you also make this mobile responsive ?

https://user-images.githubusercontent.com/75248557/124886681-1b6add00-dff2-11eb-886b-871f8a303ed4.mov

**Review by Nikhil-Ladha — COMMENTED** [review]
Looks slick 💯 , just the responsive part needs to be handled.
Also, why the mesheryctl.png is being added, I don't see it being used anywhere?

**Comment by chetak123** [comment]
Oh very sorry for that, actually initially I was just going to change image shell.png with mesheryctl.png but then found terminal directory and created terminal image from it. I will look into responsive part and remove mesheryctl.png .

**Comment by l5io** [comment]
🚀 Preview for commit 408c9a04ff1b2bd83c9a78411e5d19e240e4a0f4 at: https://60ed048c859cd56f2315e09d--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit fe5c0e26a8cd5cf26e7fc42221bd75900f7fea02 at: https://60ed1a7bdb09829230dbff2a--layer5ng.netlify.app

**Review by sudo-NithishKarthik — COMMENTED** [review]
Hmm, i don't know, doesn't the black look too contrasting, or is it only for me 🤔 ?
And, the black background is shrinking into a circle on reducing the screen size, i don't know if that was intentional or not, can you look into that please @chetak123 ? Other than that, it's pretty neat 💯 

https://user-images.githubusercontent.com/75248557/125394753-cc44f380-e3c7-11eb-83b8-8214300f8fde.mov

**Comment by l5io** [comment]
🚀 Preview for commit 772959d9c050fc374e795c2940aed692a2944591 at: https://60ed48ea599273d371fc2998--layer5ng.netlify.app

**Comment by leecalcote** [comment]
Thanks for seeing this through, @chetak123 👍

**Comment by l5io** [comment]
🚀 Preview for commit d5f41af3516c39f3667961f2185229fc93f4a172 at: https://60edeab1b58d9d485692488b--layer5ng.netlify.app

**Review by sudo-NithishKarthik — APPROVED** [review]
Looks good to me 😄  🎉 
Thank you @chetak123

File: src/components/Learn-Components/what-await-section/Features-carousel/index.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import React, {useState} from "react";
```
Comment (Nikhil-Ladha) [inline src/components/Learn-Components/what-await-section/Features-carousel/index.js:1]: Why this file is being added and where is this used?

Comment (Nikhil-Ladha) [reply src/components/Learn-Components/what-await-section/Features-carousel/index.js:1]: And it's still not responsive, is what I see 😕

Comment (sudo-NithishKarthik) [reply src/components/Learn-Components/what-await-section/Features-carousel/index.js:1]: Screenshot 2021-07-14 at 11 50 22 AM

@Nikhil-Ladha , it is responsive 🤔 , can you check again ?

Comment (chinmaym07) [reply src/components/Learn-Components/what-await-section/Features-carousel/index.js:1]: I think it's not responsive as I can see at 768px & above. I am not sure about the background color here .. as below 768px this color is getting disappeared .

image

**Comment by l5io** [comment]
🚀 Preview for commit 7b6585dd96411715c175c642db29cd1d67d39578 at: https://60f062db7c198918596bc174--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f6e5d9c9e0670bedc19422e8623aace0caa32129 at: https://60f065a0eb93cf1cdd6d7dee--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit bf6a3442583b4f79616761181833f6a8c40975e8 at: https://60f06f5c85bbdd279654c08f--layer5ng.netlify.app

**Comment by sudo-NithishKarthik** [comment]
Screenshot 2021-07-19 at 5 16 50 PM

Can u add some padding ?
The terminal is responsive now 🙌🏽 

// @Nikhil-Ladha  @chinmaym07 @Jashpatel1

**Comment by Nikhil-Ladha** [comment]
The text should be center aligned and not left.

**Comment by Nikhil-Ladha** [comment]
@chetak123 any updates here?

**Comment by chetak123** [comment]
Hey @Nikhil-Ladha can I get some more time in it as my semester exams are going on and I am little busy 😅

**Comment by Nikhil-Ladha** [comment]
Sure, any timeframe?

**Comment by chetak123** [comment]
Probably by Monday (will try to complete it as soon as possible)

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 1d59f86b158387852817d701ebcc15819209c3af at: https://6100408ff2bc8a5d53cf4039--layer5ng.netlify.app

**Pull request merged**